packages/create-sitecore-jss/src/templates/nextjs/sitecore/config/{{appName}}.config

```diff
@@ -145,19 +145,21 @@
     -->
     <layoutService>
       <configurations>
-        <config name="default">
-          <rendering>
-            <renderingContentsResolver>
+        <config name="next-default">
```

Comment:
I'm surprised this works for edge GraphQL endpoint. From what I remember, this is hard-coded to use "default". Did you try setting "default" to "true" along with this set to "false"? I'm guessing the server URL would be included in media URLs.